src/test/java/org/opensearch/testcontainers/OpenSearchContainerTest.java

```diff
@@ -59,11 +59,33 @@ public void defaultWithSecurity(final String version, final Map<String, String>
         }
     }
 
-    @DisplayName("Create OpensearchContainer with security disabled")
-    @ParameterizedTest(name = "Running Opensearch version={0} (security disabled)")
+    @DisplayName("Create OpenSearchContainer with security disabled")
+    @ParameterizedTest(name = "Running OpenSearch version={0} (security disabled)")
     @MethodSource("containers")
     public void defaultNoSecurity(final String version, final Map<String, String> env, final DockerImageName image)
             throws Exception {
+        try (OpenSearchContainer<?> container = new OpenSearchContainer<>(image).withEnv(env)) {
+            container.start();
+
+            try (RestClient client = getClient(container)) {
+                Response response = client.performRequest(new Request("GET", "/"));
+
+                assertThat(response.getStatusLine().getStatusCode(), is(200));
+                assertThat(EntityUtils.toString(response.getEntity()), containsString(version));
+
+                // No such handler should be thrown by the server
+                assertThrows(
+                        ResponseException.class,
+                        () -> client.performRequest(new Request("GET", "/_plugins/_security/health")));
+            }
+        }
+    }
+
+    @DisplayName("Create OpensearchContainer (deprecated)")
+    @ParameterizedTest(name = "Running OpenSearch version={0}")
+    @MethodSource("containers")
+    public void defaultDeprecatedName(final String version, final Map<String, String> env, final DockerImageName image)
```

Comment:
You may extract it into `OpensearchContainerTest` so when we remove it, we won't be touching this class anymore